tools/plugins/com.liferay.ide.server.ui/src/com/liferay/ide/server/ui/LiferayServerModuleLabelDecorator.java

```diff
@@ -98,4 +115,10 @@ private Image getBundleModuleImage()
 
         return image;
     }
+
+    private Image getErrorImage()
+    {
+        return ImageResource.getImage( ImageResource.IMG_PROJECT_MISSING );
```

Comment:
Yes we can do that.  Can you make this an "Error" image and on the bottom
left of the module?

On Wed, Aug 10, 2016 at 7:14 AM, Terry Jia <notifications@github.com> wrote:

> In tools/plugins/com.liferay.ide.server.ui/src/com/liferay/ide/server/ui/
> LiferayServerModuleLabelDecorator.java
> <https://github.com/gamerson/liferay-ide/pull/1693#discussion_r74232360>:
>
> > @@ -98,4 +115,10 @@ private Image getBundleModuleImage()
> >
> >          return image;
> >      }
> > +
> > +    private Image getErrorImage()
> > +    {
> > +        return ImageResource.getImage( ImageResource.IMG_PROJECT_MISSING );
>
> Can I open another ticket for status text?
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub
> <https://github.com/gamerson/liferay-ide/pull/1693/files/00ab10c1e64525a362f5ad933bbb0d134001c4f5#r74232360>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAkVFYrWvCqDMFNZAAPz2y7vC2QdkjIhks5qecCkgaJpZM4Jfwho>
> .
>



-- 
Greg Amerson
Liferay Developer Tools
Liferay, Inc. www.liferay.com